app/components/new-sign-up/get-started.hbs

```diff
@@ -6,20 +6,20 @@
   <h3 class="landing__sub-heading" data-test-subHeading>
     {{this.subHeading}}
   </h3>
-  {{#if (eq @state "get-started")}}
+  {{#if (eq @currentStep "get-started")}}
     <ul class="landing__features-lists">
       <li class="landing__features-list" data-test-li1>Use Features</li>
       <li class="landing__features-list" data-test-li2>Display yourself on the (members) page</li>
     </ul>
   {{/if}}
 
   <div class="get-started__btn">
-    <SignUp::Button
-      @clickHandler={{@changeRouteParams}}
-      @state={{@state}}
+    <NewSignUp::Button
+      @clickHandler={{@handleRouteParamsChange}}
       @disabled={{false}}
+      @isSubmitClicked={{@isSubmitClicked}}
     >
-      {{if (eq @state "get-started") "Get Started" "Let's Go"}}
-    </SignUp::Button>
+      {{if (eq @currentStep "get-started") "Get Started" "Let's Go"}}
```

Comment:
Messaging shoud ideally come from constants, so they can be maintained with relative ease later.